ansible/bootstrap.yml

```
@@ -1,5 +1,15 @@
 ---
 
+- hosts: localhost
+  gather_facts: false
+  become: false
+  tags:
+    - depreciated
+  tasks:
+    - debug:
+        msg: "Variables prefixed secrets_openhpc_* are depreciated and should be replaced with vault_*, please see ansible/roles/passwords/defaults/main.yml"
```

```suggestion
        msg: "Variables prefixed secrets_openhpc_* are deprecated and should be replaced with vault_*, please see ansible/roles/passwords/defaults/main.yml"
```
Is it worth failing and advising to re-run the password generation script?

---

If having to rerun that is OK, why do we need to support secrets_openhpc_* at all? I thought the idea was to support updating the appliance while using the existing environment's secrets file.

---

Running the password generation script should rename the variables for you. If you don't run it, then you will still have, for example, ``secrets_openhpc_grafana_admin_password`` and not ``vault_grafana_admin_password`` defined (I think, unless I missed something).

---

Ah, I'd missed the fact that wouldn't regenerate the actual secrets, that's neat.